### PR TITLE
Activate B015 to detect pointless comparison

### PR DESCRIPTION
**Pull request by hussein-awala**
Oups, 32 ignored checks in our tests.

### CODE REVIEW

**Review by Taragolis — APPROVED** [review]
Yeah! More bugbear checks!

**Comment by Taragolis** [comment]
> Oups, 32 ignored checks in our tests.

Whoops some of them seem broken for a while or just some change in codebase not reflected or we miss some regression found

**Review by dirrao — APPROVED** [review]

**Comment by potiuk** [comment]
Yeah. Most of them comparing tuples to lists :) but a few require closer look.

I like some of those ruff checks :)

File: tests/models/test_baseoperator.py [hunk 1]
```diff
@@ -638,25 +638,25 @@ def test_lineage_composition(self):
         task1.supports_lineage = True
 
         # note: operator precedence still applies
-        inlet > task1 | (task2 > outlet)
+        inlet > task1 | (task2 > outlet)  # noqa: B015
```
Comment (uranusjr) [inline tests/models/test_baseoperator.py:641]: What does this line do?

**Review by vincbeck — APPROVED** [review]

**Review by eladkal — APPROVED** [review]

**Pull request closed without merging**